tests/openwisp2/sample_notifications/templates/admin/base.html

```diff
@@ -0,0 +1,11 @@
+{% extends "admin/base.html" %}
```

Comment:
Is this really needed? I would have expected this template to be found by the DepdendencyLoader in openwisp-utils.

If this template is optional and is here only for example purposes, please state it both here (with a django template comment) and in the README.

Reply:
The tests fails without it. As I mentioned on IM: 

> The base template we use for admin site has URL for `openwisp_notifications`. 


Some tests in `test_admin.py` fails if this is not present, since Django is unable to resolve that URL.